compiler/luci/lang/include/luci/IR/CircleNodes.lst

```diff
@@ -125,6 +126,7 @@ CIRCLE_NODE(BCQ_FULLY_CONNECTED, luci::CircleBCQFullyConnected)
 CIRCLE_NODE(BCQ_GATHER, luci::CircleBCQGather)
 CIRCLE_NODE(INSTANCE_NORM, luci::CircleInstanceNorm)
 // Virtual node(s)
+CIRCLE_NODE(CIRCLEBIDIRECTIONAL_SEQUENCE_LSTMOUT, luci::CircleBidirectionalSequenceLSTMOut)
```

Comment:
here too

Reply:
can you please use `CIRCLEBIDIRECTIONAL_SEQUENCE_LSTM_OUT` ?

Reply:
will address this, thanks.